app/pages/_components.py

```
@@ -0,0 +1,300 @@
+# ---------------------------------------------------------------------------
+
+def delivery_card(
+    badge: str,
+    title: str,
+    description: str,
+    price: str,
+    price_note: str,
+    features: list[str],
+    cta_label: str,
+    cta_href: str,
+    featured: bool = False,
```

Ruff (0.15.5)

[error] Boolean-typed positional argument in function definition

(FBT001)